src/action/mod.rs

```diff
@@ -23,6 +23,7 @@ pub mod interfaces;
 
 pub mod metadata;
 pub mod startup;
+pub mod list_directory;
```

Comment:
I could have sworn that I left a comment about renaming the module, but I cannot find it. So far I tried to make all action names one-word identifiers, so lets try to keep this tradition for now. Something like `listdir` would work fine, I think.

Reply:
Done.